diff -Naur standalone/LicenseCheck/InvalidLicense/InvalidLicense-0.ebuild fixed/LicenseCheck/InvalidLicense/InvalidLicense-0.ebuild
--- standalone/LicenseCheck/InvalidLicense/InvalidLicense-0.ebuild
+++ fixed/LicenseCheck/InvalidLicense/InvalidLicense-0.ebuild
@@ -1,4 +1,4 @@
 DESCRIPTION="Ebuild with invalid LICENSE"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-LICENSE="( )"
+LICENSE="BSD"
 SLOT="0"
diff -Naur standalone/LicenseCheck/InvalidLicense/InvalidLicense-1.ebuild fixed/LicenseCheck/InvalidLicense/InvalidLicense-1.ebuild
--- standalone/LicenseCheck/InvalidLicense/InvalidLicense-1.ebuild
+++ fixed/LicenseCheck/InvalidLicense/InvalidLicense-1.ebuild
@@ -1,4 +1,4 @@
 DESCRIPTION="Ebuild with invalid LICENSE"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-LICENSE="?"
+LICENSE="BSD"
 SLOT="0"
diff -Naur standalone/LicenseCheck/InvalidLicense/InvalidLicense-2.ebuild fixed/LicenseCheck/InvalidLicense/InvalidLicense-2.ebuild
--- standalone/LicenseCheck/InvalidLicense/InvalidLicense-2.ebuild
+++ fixed/LicenseCheck/InvalidLicense/InvalidLicense-2.ebuild
@@ -1,5 +1,5 @@
 DESCRIPTION="Ebuild with invalid LICENSE"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-LICENSE="used?"
+LICENSE="used? ( BSD )"
 SLOT="0"
 IUSE="used"
