diff -Naur standalone/RequiredUseCheck/UnstatedIuse/UnstatedIuse-0.ebuild fixed/RequiredUseCheck/UnstatedIuse/UnstatedIuse-0.ebuild
--- standalone/RequiredUseCheck/UnstatedIuse/UnstatedIuse-0.ebuild	2019-11-23 13:50:52.531217164 -0700
+++ fixed/RequiredUseCheck/UnstatedIuse/UnstatedIuse-0.ebuild	2019-11-23 13:51:47.992485035 -0700
@@ -4,5 +4,5 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-IUSE="required"
+IUSE="required used"
 REQUIRED_USE="|| ( required used )"
