diff -Naur standalone/InsintoCheck/DeprecatedInsinto/DeprecatedInsinto-1.ebuild fixed/InsintoCheck/DeprecatedInsinto/DeprecatedInsinto-1.ebuild
--- standalone/InsintoCheck/DeprecatedInsinto/DeprecatedInsinto-1.ebuild	2020-01-21 13:20:18.815286720 -0700
+++ fixed/InsintoCheck/DeprecatedInsinto/DeprecatedInsinto-1.ebuild	2020-01-21 13:23:51.965234839 -0700
@@ -5,14 +5,11 @@
 LICENSE="BSD"
 
 src_install() {
-	insinto /usr/share/doc/${PF}
-	doins foo
-	insinto /usr/share/doc/"${PF}"
-	doins bar
-	insinto /usr/share/doc/${PF}/
-	doins -r html
-	insinto /usr/share/doc/${PF}/examples
-	doins samples/*
-	insinto /usr/share/doc/"${PF}"/examples
-	doins foo/examples/*
+	dodoc foo
+	dodoc bar
+	dodoc -r html
+
+	docinto examples
+	dodoc samples/*
+	dodoc foo/examples/*
 }
