diff -Naur standalone/IuseMetadataCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild fixed/IuseMetadataCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild
--- standalone/IuseMetadataCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild	2019-09-18 22:05:56.510227607 -0600
+++ fixed/IuseMetadataCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild	2019-09-18 22:07:16.121549184 -0600
@@ -3,4 +3,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 SLOT="0"
 LICENSE="BSD"
-IUSE="amd64 used flag1 +flag2 -flag3"
+IUSE="used flag1 +flag2 -flag3"
diff -Naur standalone/IuseMetadataCheck/UnknownUseFlags/metadata.xml fixed/IuseMetadataCheck/UnknownUseFlags/metadata.xml
--- standalone/IuseMetadataCheck/UnknownUseFlags/metadata.xml	2019-09-18 21:53:02.093097947 -0600
+++ fixed/IuseMetadataCheck/UnknownUseFlags/metadata.xml	2019-09-18 22:07:31.287610443 -0600
@@ -7,5 +7,6 @@
 	<use>
 		<flag name="flag1">USE flag 1</flag>
 		<flag name="flag2">USE flag 2</flag>
+		<flag name="flag3">USE flag 3</flag>
 	</use>
 </pkgmetadata>
