diff -Naur standalone/VariableScopeCheck/EbuildVariableScope/EbuildVariableScope-0.ebuild fixed/VariableScopeCheck/EbuildVariableScope/EbuildVariableScope-0.ebuild
--- standalone/VariableScopeCheck/EbuildVariableScope/EbuildVariableScope-0.ebuild	2021-03-17 01:20:05.423678951 -0600
+++ fixed/VariableScopeCheck/EbuildVariableScope/EbuildVariableScope-0.ebuild	2021-03-17 01:21:57.421132212 -0600
@@ -5,11 +5,9 @@ HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"

-DOC_CONTENTS="Hello ${ROOT}"
-
 src_configure() {
 	# EROOT isn't allowed in src_* phases
-	econf --sysconfdir="${EROOT}/etc/${PN}"
+	econf --sysconfdir="${EPREFIX}/etc/${PN}"
 }

 pkg_postinst() {
