diff -Naur standalone/VisibilityCheck/NonexistentDeps/NonexistentDeps-0.ebuild fixed/VisibilityCheck/NonexistentDeps/NonexistentDeps-0.ebuild
--- standalone/VisibilityCheck/NonexistentDeps/NonexistentDeps-0.ebuild	2019-10-30 18:27:24.954171041 -0600
+++ fixed/VisibilityCheck/NonexistentDeps/NonexistentDeps-0.ebuild	2019-10-30 18:31:12.925907712 -0600
@@ -2,4 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-DEPEND="|| ( stub/stub1 stub/stub2 cat/nonexistent )"
+DEPEND="|| ( stub/stub1 stub/stub2 )"
