diff -Naur standalone/RestrictCheck/UnknownRestrict/UnknownRestrict-0.ebuild fixed/RestrictCheck/UnknownRestrict/UnknownRestrict-0.ebuild
--- standalone/RestrictCheck/UnknownRestrict/UnknownRestrict-0.ebuild	2019-10-08 02:36:30.390654096 -0600
+++ fixed/RestrictCheck/UnknownRestrict/UnknownRestrict-0.ebuild	2019-10-08 02:42:33.587430292 -0600
@@ -2,4 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-RESTRICT="foo"
+RESTRICT="mirror"
