diff -Naur standalone/EbuildReservedCheck/EbuildReservedName/EbuildReservedName-0.ebuild fixed/EbuildReservedCheck/EbuildReservedName/EbuildReservedName-0.ebuild
--- standalone/EbuildReservedCheck/EbuildReservedName/EbuildReservedName-0.ebuild
+++ fixed/EbuildReservedCheck/EbuildReservedName/EbuildReservedName-0.ebuild
@@ -3,20 +3,19 @@ HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"

-prepare_locale() {
-	DYNAMIC_DEPS="2"
-	_hook_prepare="3"
+my_prepare_locale() {
+	MY_DYNAMIC_DEPS="2"
+	_my_prepare="3"
 }

-__ORIG_CC="STUB"
+MY_ORIG_CC="STUB"
 EBUILD_SUCCESS_HOOKS="true"
-EBUILD_TEST="1"
 REBUILD_ALL="1"

-post_src_unpack() {
+my_post_src_unpack() {
 	echo "Larry was here"
 }

-pre_src_test() {
+my_pre_src_test() {
 	echo "Larry was even here"
 }
