diff -Naur standalone/RestrictTestCheck/MissingTestRestrict/MissingTestRestrict-0.ebuild fixed/RestrictTestCheck/MissingTestRestrict/MissingTestRestrict-0.ebuild
--- standalone/RestrictTestCheck/MissingTestRestrict/MissingTestRestrict-0.ebuild	2019-11-23 14:07:17.053630938 -0700
+++ fixed/RestrictTestCheck/MissingTestRestrict/MissingTestRestrict-0.ebuild	2019-11-23 14:09:17.952162861 -0700
@@ -3,3 +3,4 @@
 LICENSE="BSD"
 SLOT="0"
 IUSE="test"
+RESTRICT="!test? ( test )"
