--- gentoo/UseFlagsWithoutEffectsCheck/UseFlagWithoutDeps/UseFlagWithoutDeps-2.ebuild
+++ fixed/UseFlagsWithoutEffectsCheck/UseFlagWithoutDeps/UseFlagWithoutDeps-2.ebuild
@@ -8,8 +8,8 @@ HOMEPAGE="https://github.com/pkgcore/pkgcheck"

 LICENSE="BSD"
 SLOT="0"
-IUSE="bash-completion ipv6"
+IUSE="ipv6"

 src_compile() {
-	emake IPV6=$(use ipv6) BASH="$(use bash-completion)"
+	emake IPV6=$(use ipv6) BASH="1"
 }
