diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild	2021-03-04 00:28:34.661859223 -0700
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild	2021-03-18 16:08:58.428916388 -0600
@@ -1,5 +1,5 @@
 DESCRIPTION="Ebuild with variable in HOMEPAGE"
-HOMEPAGE="https://github.com/pkgcore/${PN}"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 # commented lines that would otherwise be flagged are ignored
 #HOMEPAGE="https://github.com/${P}"
 SLOT="0"
diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild	2021-03-18 16:08:15.425975991 -0600
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild	2021-03-18 16:09:08.487136344 -0600
@@ -2,5 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 SLOT="0"
 LICENSE="BSD"
-ARCH="~x86"
-KEYWORDS="~amd64 ${ARCH}"
+KEYWORDS="~amd64 ~x86"
