diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild	2021-03-04 00:28:34.661859223 -0700
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-0.ebuild	2021-03-18 16:08:58.428916388 -0600
@@ -1,5 +1,5 @@
 DESCRIPTION="Ebuild with variable in HOMEPAGE"
-HOMEPAGE="https://github.com/pkgcore/${PN}"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 # commented lines that would otherwise be flagged are ignored
 #HOMEPAGE="https://github.com/${P}"
 SLOT="0"
diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild	2021-03-18 16:08:15.425975991 -0600
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-1.ebuild	2021-03-18 16:09:08.487136344 -0600
@@ -2,5 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-ARCH="~x86"
-KEYWORDS="~amd64 ${ARCH}"
+KEYWORDS="~amd64 ~x86"
diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-2.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-2.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-2.ebuild	2022-04-30 20:58:05.075839682 +0200
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-2.ebuild.fixed	2022-04-30 20:57:57.498766323 +0200
@@ -1,7 +1,6 @@
 DESCRIPTION="Ebuild with variable in LICENSE"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-MY_LICENSE="BSD"
-LICENSE="${MY_LICENSE}"
+LICENSE="BSD"
 LICENSE="${LICENSE} BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-3.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-3.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-3.ebuild	2022-04-30 20:58:05.075839682 +0200
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-3.ebuild.fixed	2022-04-30 20:57:57.498766323 +0200
@@ -1,7 +1,6 @@
 DESCRIPTION="Ebuild with variable in LICENSE"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-MY_LICENSE="BSD"
 LICENSE=""
-LICENSE+="${MY_LICENSE}"
+LICENSE+="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
diff -Naur standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-4.ebuild fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-4.ebuild
--- standalone/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-4.ebuild	2022-04-30 20:58:05.075839682 +0200
+++ fixed/MetadataVarCheck/ReferenceInMetadataVar/ReferenceInMetadataVar-4.ebuild.fixed	2022-04-30 20:57:57.498766323 +0200
@@ -2,5 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
-LICENSE="${LICENSE} ${LICENSE/B/B}"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
