diff -Naur standalone/RestrictCheck/InvalidRestrict/InvalidRestrict-0.ebuild fixed/RestrictCheck/InvalidRestrict/InvalidRestrict-0.ebuild
--- standalone/RestrictCheck/InvalidRestrict/InvalidRestrict-0.ebuild	2019-11-23 15:19:40.247982566 -0700
+++ fixed/RestrictCheck/InvalidRestrict/InvalidRestrict-0.ebuild	2019-11-23 15:20:02.882083351 -0700
@@ -3,4 +3,4 @@
 LICENSE="BSD"
 SLOT="0"
 IUSE="used"
-RESTRICT="!used ( test )"
+RESTRICT="!used? ( test )"
