diff -Naur standalone/SrcUriCheck/BadFilename/BadFilename-0.ebuild fixed/SrcUriCheck/BadFilename/BadFilename-0.ebuild
--- standalone/SrcUriCheck/BadFilename/BadFilename-0.ebuild	2019-09-07 06:28:49.253085735 -0600
+++ fixed/SrcUriCheck/BadFilename/BadFilename-0.ebuild	2019-09-07 06:34:24.603430950 -0600
@@ -1,9 +1,10 @@
+EAPI=2
 DESCRIPTION="Ebuild with unspecific SRC_URI filenames"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 SRC_URI="
-	https://github.com/pkgcore/pkgcheck/${PN}.tar.gz
-	https://github.com/pkgcore/pkgcheck/${PV}.tar.gz
-	https://github.com/pkgcore/pkgcheck/v${PV}.tar.gz
+	https://github.com/pkgcore/pkgcheck/${PN}.tar.gz -> ${P}.tar.gz
+	https://github.com/pkgcore/pkgcheck/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/pkgcore/pkgcheck/v${PV}.tar.gz -> ${P}.tar.gz
 "
 LICENSE="BSD"
 SLOT="0"
diff -Naur standalone/SrcUriCheck/BadFilename/Manifest fixed/SrcUriCheck/BadFilename/Manifest
--- standalone/SrcUriCheck/BadFilename/Manifest	2019-09-07 06:29:08.537163522 -0600
+++ fixed/SrcUriCheck/BadFilename/Manifest	2019-09-07 06:33:35.393236829 -0600
@@ -1,3 +1 @@
-DIST BadFilename.tar.gz 153310 BLAKE2B b7484cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7e45bcb7b255cdf810ce8cd16a957fb5bca3d1e71c088cd894968641db5dfae1c4c059df836 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
-DIST 0.tar.gz 153310 BLAKE2B b7484cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7945bcb7b255cdf810ceecd16a957fb5bca3d1e71c088cd894968641db5dfae1c4c059df836 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
-DIST v0.tar.gz 153310 BLAKE2B b7484cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7945bcb7b255cdf810ce8cd16a957fe5bca3d1e71c088cd894968641db5dfae1c4c059df836 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
+DIST BadFilename-0.tar.gz 153310 BLAKE2B b7484cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7945bcb7b255cdf810ce8cd16a957fb5bca3d1e71c088cd894968641db5dfae1c4c059df836 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
