diff -Naur eclass/eclass/missing-docs.eclass fixed/eclass/missing-docs.eclass
--- eclass/eclass/missing-docs.eclass	2020-11-24 02:09:50.259835420 -0700
+++ fixed/eclass/missing-docs.eclass	2020-11-24 02:12:01.756701701 -0700
@@ -10,6 +10,10 @@
 # Public documented stub function.
 missing-docs_documented_func() { :; }
 
+# @FUNCTION: missing-docs_public_func
+# @USAGE:
+# @DESCRIPTION:
+# Public stub function.
 missing-docs_public_func() { :; }
 
 MISSING_DOCS_PUBLIC_VAR=""
