diff -Naur visibility/DeprecatedPkg/nonoptional/nonoptional-0.ebuild fixed/DeprecatedPkg/nonoptional/nonoptional-0.ebuild
--- visibility/DeprecatedPkg/nonoptional/nonoptional-0.ebuild	2019-11-30 04:17:38.664181249 -0700
+++ fixed/DeprecatedPkg/nonoptional/nonoptional-0.ebuild	2019-11-30 04:26:06.206411351 -0700
@@ -4,4 +4,4 @@
 SLOT="0"
 LICENSE="BSD"
 KEYWORDS="~amd64"
-RDEPEND="stub/deprecated"
+RDEPEND="stub/unstable"
diff -Naur visibility/DeprecatedPkg/optional/optional-0.ebuild fixed/DeprecatedPkg/optional/optional-0.ebuild
--- visibility/DeprecatedPkg/optional/optional-0.ebuild	2019-11-30 04:17:38.664181249 -0700
+++ fixed/DeprecatedPkg/optional/optional-0.ebuild	2019-11-30 04:26:17.882459004 -0700
@@ -4,4 +4,4 @@
 SLOT="0"
 LICENSE="BSD"
 KEYWORDS="~amd64"
-RDEPEND="|| ( stub/unstable stub/deprecated )"
+RDEPEND="stub/unstable"
