diff -Naur standalone/SrcUriCheck/BadProtocol/BadProtocol-0.ebuild fixed/SrcUriCheck/BadProtocol/BadProtocol-0.ebuild
--- standalone/SrcUriCheck/BadProtocol/BadProtocol-0.ebuild	2019-08-23 16:40:09.597584899 -0600
+++ fixed/SrcUriCheck/BadProtocol/BadProtocol-0.ebuild	2019-08-23 16:43:17.724321149 -0600
@@ -1,8 +1,5 @@
 DESCRIPTION="Ebuild with SRC_URI using an unsupported protocol"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-SRC_URI="
-	htps://github.com/pkgcore/pkgcheck/${P}.tar.gz
-	gopher://github.com/pkgcore/pkgcheck/${P}.tar.gz
-"
+SRC_URI="https://github.com/pkgcore/pkgcheck/${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
