diff -Naur standalone/DeclarationShadowedCheck/VariableShadowed/VariableShadowed-0.ebuild fixed/DeclarationShadowedCheck/VariableShadowed/VariableShadowed-0.ebuild
--- standalone/DeclarationShadowedCheck/VariableShadowed/VariableShadowed-0.ebuild
+++ fixed/DeclarationShadowedCheck/VariableShadowed/VariableShadowed-0.ebuild
@@ -2,15 +2,12 @@ HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 S=${WORKDIR}

-VAL=
-
 LICENSE="BSD"
 SLOT="0"
-RESTRICT="!test? ( test )"
+RESTRICT="test"

 RDEPEND="dev-lang/ruby"
 DEPEND="${RDEPEND}"
-RDEPEND="dev-ruby/stub"
+RDEPEND+="dev-ruby/stub"

-RESTRICT="test"
 VAL=5
