diff -Naur eclass/eclass/missing.eclass fixed/eclass/missing.eclass
--- eclass/eclass/missing.eclass	2020-11-24 01:48:04.220381235 -0700
+++ fixed/eclass/missing.eclass	2020-11-24 01:48:43.302231824 -0700
@@ -4,6 +4,10 @@
 # @SUPPORTED_EAPIS: 6 7
 # @BLURB: Stub eclass with missing function and variable documentation.
 
+# @FUNCTION: missing-docs_public_function
+# @USAGE:
+# @DESCRIPTION:
+# Public stub function.
 missing-docs_public_function() { :; }
 
 MISSING_DOCS_PUBLIC_VAR=""
