diff -Naur standalone/IuseCheck/InvalidUseFlags/InvalidUseFlags-0.ebuild fixed/IuseCheck/InvalidUseFlags/InvalidUseFlags-0.ebuild
--- standalone/IuseCheck/InvalidUseFlags/InvalidUseFlags-0.ebuild	2021-05-24 04:15:26.694190651 -0600
+++ fixed/IuseCheck/InvalidUseFlags/InvalidUseFlags-0.ebuild	2021-05-24 04:26:29.092543549 -0600
@@ -2,4 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-IUSE="used +flag1 -flag2 + -"
+IUSE="used flag1 flag2"
diff -Naur standalone/IuseCheck/InvalidUseFlags/InvalidUseFlags-4.ebuild fixed/IuseCheck/InvalidUseFlags/InvalidUseFlags-4.ebuild
--- standalone/IuseCheck/InvalidUseFlags/InvalidUseFlags-4.ebuild	2021-05-24 04:15:45.921607273 -0600
+++ fixed/IuseCheck/InvalidUseFlags/InvalidUseFlags-4.ebuild	2021-05-24 04:26:36.519704480 -0600
@@ -4,4 +4,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-IUSE="used +flag1 +flag2 + -"
+IUSE="used +flag1 +flag2"
