diff -Naur standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-0.ebuild fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-0.ebuild
--- standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-0.ebuild	2019-11-23 13:38:24.756495493 -0700
+++ fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-0.ebuild	2019-11-23 13:40:55.077141263 -0700
@@ -3,4 +3,5 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-REQUIRED_USE="|| ( )"
+IUSE="required used"
+REQUIRED_USE="|| ( required used )"
diff -Naur standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-1.ebuild fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-1.ebuild
--- standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-1.ebuild	2019-11-23 13:33:43.839154284 -0700
+++ fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-1.ebuild	2019-11-23 13:41:17.866239168 -0700
@@ -3,4 +3,5 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-REQUIRED_USE="?"
+IUSE="required used"
+REQUIRED_USE="used? ( required )"
diff -Naur standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-2.ebuild fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-2.ebuild
--- standalone/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-2.ebuild	2019-11-23 13:38:59.669645478 -0700
+++ fixed/RequiredUseCheck/InvalidRequiredUse/InvalidRequiredUse-2.ebuild	2019-11-23 13:41:31.418297387 -0700
@@ -3,5 +3,5 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-IUSE="required"
-REQUIRED_USE="required?"
+IUSE="required used"
+REQUIRED_USE="required? ( used )"
