diff --git stable_keywords/StableKeywordsCheck/StableKeywords/StableKeywords-0.ebuild fixed/StableKeywordsCheck/StableKeywords/StableKeywords-0.ebuild
index fc606dee..52f19ba0 100644
--- stable_keywords/StableKeywordsCheck/StableKeywords/StableKeywords-0.ebuild
+++ fixed/StableKeywordsCheck/StableKeywords/StableKeywords-0.ebuild
@@ -9,4 +9,4 @@ SRC_URI="https://example.com/"
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="amd64"
+KEYWORDS="~amd64"
diff --git stable_keywords/StableKeywordsCheck/StableKeywords/StableKeywords-1.ebuild fixed/StableKeywordsCheck/StableKeywords/StableKeywords-1.ebuild
index e8774608..12e0329a 100644
--- stable_keywords/StableKeywordsCheck/StableKeywords/StableKeywords-1.ebuild
+++ fixed/StableKeywordsCheck/StableKeywords/StableKeywords-1.ebuild
@@ -9,4 +9,4 @@ SRC_URI="https://example.com/"
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="~amd64 x86"
+KEYWORDS="~amd64 ~x86"
