diff -Naur profiledir/profiles/default/bad_parent/parent fixed/profiles/default/bad_parent/parent
--- profiledir/profiles/default/bad_parent/parent	2019-11-11 18:59:21.638234845 -0700
+++ fixed/profiles/default/bad_parent/parent	2019-11-12 20:45:01.973797244 -0700
@@ -1,2 +1 @@
-../nonexistent
 ..
diff -Naur profiledir/profiles/deprecated/nonexistent/deprecated fixed/profiles/deprecated/nonexistent/deprecated
--- profiledir/profiles/deprecated/nonexistent/deprecated	2019-11-12 20:35:43.195830784 -0700
+++ fixed/profiles/deprecated/nonexistent/deprecated	2019-11-12 20:45:16.582848028 -0700
@@ -1 +1 @@
-default/nonexistent
+default
