diff -Naur standalone/IuseCheck/UnknownUseFlags/metadata.xml fixed/IuseCheck/UnknownUseFlags/metadata.xml
--- standalone/IuseCheck/UnknownUseFlags/metadata.xml	2021-03-24 15:23:53.617079961 -0600
+++ fixed/IuseCheck/UnknownUseFlags/metadata.xml	2021-05-24 04:22:38.816553926 -0600
@@ -7,5 +7,6 @@
 	<use>
 		<flag name="flag1">USE flag 1</flag>
 		<flag name="flag2">USE flag 2</flag>
+		<flag name="flag3">USE flag 3</flag>
 	</use>
 </pkgmetadata>
diff -Naur standalone/IuseCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild fixed/IuseCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild
--- standalone/IuseCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild	2021-05-24 04:16:22.701404227 -0600
+++ fixed/IuseCheck/UnknownUseFlags/UnknownUseFlags-0.ebuild	2021-05-24 04:22:44.096668337 -0600
@@ -4,4 +4,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-IUSE="amd64 used flag1 +flag2 +flag3"
+IUSE="used flag1 +flag2 +flag3"
