diff -dupr python/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-0.ebuild fixed/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-0.ebuild
index face1460..d8f40dd7 100644
--- python/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-0.ebuild
+++ fixed/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-0.ebuild
@@ -21,5 +21,5 @@ BDEPEND="${PYTHON_DEPS}
 "

 python_check_deps() {
-	has_version -b "dev-python/lxml[${PYTHON_USEDEP}]"
+	python_has_version "dev-python/lxml[${PYTHON_USEDEP}]"
 }
diff -dupr python/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-1.ebuild fixed/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-1.ebuild
index 240f5364..48f4ae11 100644
--- python/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-1.ebuild
+++ fixed/PythonCheck/PythonHasVersionUsage/PythonHasVersionUsage-1.ebuild
@@ -22,6 +22,6 @@ BDEPEND="${PYTHON_DEPS}
 "

 python_check_deps() {
-	has_version -b "dev-python/lxml[${PYTHON_USEDEP}]" &&
-	has_version -b "dev-python/gpep517[${PYTHON_USEDEP}]"
+	python_has_version "dev-python/lxml[${PYTHON_USEDEP}]" &&
+	python_has_version "dev-python/gpep517[${PYTHON_USEDEP}]"
 }
diff -dupr a/python/PythonCheck/PythonHasVersionMissingPythonUseDep/PythonHasVersionMissingPythonUseDep-0.ebuild b/python/PythonCheck/PythonHasVersionMissingPythonUseDep/PythonHasVersionMissingPythonUseDep-0.ebuild
index 24ebb878..48f4ae11 100644
--- python/PythonCheck/PythonHasVersionMissingPythonUseDep/PythonHasVersionMissingPythonUseDep-0.ebuild
+++ fixed/PythonCheck/PythonHasVersionMissingPythonUseDep/PythonHasVersionMissingPythonUseDep-0.ebuild
@@ -22,6 +22,6 @@ BDEPEND="${PYTHON_DEPS}
 "

 python_check_deps() {
-	has_version -b "dev-python/lxml" &&
-	python_has_version "dev-python/gpep517"
+	python_has_version "dev-python/lxml[${PYTHON_USEDEP}]" &&
+	python_has_version "dev-python/gpep517[${PYTHON_USEDEP}]"
 }
diff -dupr python/PythonCheck/PythonAnyMismatchedDepHasVersionCheck/PythonAnyMismatchedDepHasVersionCheck-0.ebuild fixed/PythonCheck/PythonAnyMismatchedDepHasVersionCheck/PythonAnyMismatchedDepHasVersionCheck-0.ebuild
index a70a4d3e..21db4ebf 100644
--- python/PythonCheck/PythonAnyMismatchedDepHasVersionCheck/PythonAnyMismatchedDepHasVersionCheck-0.ebuild
+++ fixed/PythonCheck/PythonAnyMismatchedDepHasVersionCheck/PythonAnyMismatchedDepHasVersionCheck-0.ebuild
@@ -32,8 +32,7 @@ BDEPEND="${PYTHON_DEPS}

 python_check_deps() {
 	use test || return 1
-	has_version "dev-python/lxml[${PYTHON_USEDEP}]" &&
-	has_version -r "dev-python/lxml[${PYTHON_USEDEP}]" &&
+	python_has_version "dev-python/lxml[${PYTHON_USEDEP}]" &&
 	python_has_version "dev-python/gpep517[${PYTHON_USEDEP}]" &&
-	python_has_version "dev-python/flit_core[${PYTHON_USEDEP}]"
+	python_has_version -d "dev-python/flit_core[${PYTHON_USEDEP}]"
 }
