diff -Naur python/PythonCheck/PythonMissingDeps/PythonMissingDeps-0.ebuild fixed/PythonCheck/PythonMissingDeps/PythonMissingDeps-0.ebuild
--- python/PythonCheck/PythonMissingDeps/PythonMissingDeps-0.ebuild	2020-12-01 12:19:27.856851956 -0700
+++ fixed/PythonCheck/PythonMissingDeps/PythonMissingDeps-0.ebuild	2020-12-01 12:20:13.815852958 -0700
@@ -8,3 +8,5 @@
 LICENSE="BSD"
 SLOT="0"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}"
--- python/PythonCheck/PythonMissingDeps/PythonMissingDeps-1.ebuild
+++ fixed/PythonCheck/PythonMissingDeps/PythonMissingDeps-1.ebuild
@@ -15,3 +15,4 @@
 IUSE="python"
 
 RDEPEND="python? ( ${PYTHON_DEPS} )"
+BDEPEND="python? ( ${DISTUTILS_DEPS} )"
