diff -Naur standalone/DependencyCheck/InvalidDependency/InvalidDependency-0.ebuild fixed/DependencyCheck/InvalidDependency/InvalidDependency-0.ebuild
--- standalone/DependencyCheck/InvalidDependency/InvalidDependency-0.ebuild	2019-11-15 00:48:27.544231225 -0700
+++ fixed/DependencyCheck/InvalidDependency/InvalidDependency-0.ebuild	2019-11-15 00:49:40.598525509 -0700
@@ -2,6 +2,6 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 SLOT="0"
 LICENSE="BSD"
-DEPEND="pkg1"
-RDEPEND="pkg2"
-PDEPEND="pkg3"
+DEPEND="cat/pkg1"
+RDEPEND="cat/pkg2"
+PDEPEND="cat/pkg3"
