diff -Naur standalone/SourcingCheck/InvalidEapi/InvalidEapi-0.ebuild fixed/SourcingCheck/InvalidEapi/InvalidEapi-0.ebuild
--- standalone/SourcingCheck/InvalidEapi/InvalidEapi-0.ebuild
+++ fixed/SourcingCheck/InvalidEapi/InvalidEapi-0.ebuild
@@ -1,4 +1,4 @@
-EAPI=9999
+EAPI=7
 DESCRIPTION="Ebuild using unsupported EAPI"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
diff -Naur standalone/SourcingCheck/InvalidEapi/InvalidEapi-1.ebuild fixed/SourcingCheck/InvalidEapi/InvalidEapi-1.ebuild
--- standalone/SourcingCheck/InvalidEapi/InvalidEapi-1.ebuild
+++ fixed/SourcingCheck/InvalidEapi/InvalidEapi-1.ebuild
@@ -1,4 +1,4 @@
-EAPI="invalid!"
+EAPI=7
 DESCRIPTION="Ebuild using invalid EAPI"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
