diff -Naur standalone/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-0.ebuild fixed/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-0.ebuild
--- standalone/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-0.ebuild	2019-10-09 06:27:05.229866513 -0600
+++ fixed/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-0.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,5 +0,0 @@
-DESCRIPTION="Ebuild with dropped keywords"
-HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-LICENSE="BSD"
-SLOT="0"
-KEYWORDS="~amd64 ~x86"
diff -Naur standalone/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-1.ebuild fixed/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-1.ebuild
--- standalone/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-1.ebuild	2019-10-09 06:27:06.151870892 -0600
+++ fixed/DroppedKeywordsCheck/DroppedKeywords/DroppedKeywords-1.ebuild	2019-10-09 06:27:27.411971799 -0600
@@ -2,4 +2,4 @@
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64"
+KEYWORDS="~amd64 ~x86"
