diff -Naur standalone/EclassManualDepsCheck/RubyMissingDeps/RubyMissingDeps-1.ebuild fixed/EclassManualDepsCheck/RubyMissingDeps/RubyMissingDeps-1.ebuild
--- standalone/EclassManualDepsCheck/RubyMissingDeps/RubyMissingDeps-1.ebuild
+++ fixed/EclassManualDepsCheck/RubyMissingDeps/RubyMissingDeps-1.ebuild
@@ -6,3 +6,5 @@ DESCRIPTION="Optional inherit without dep"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
 LICENSE="BSD"
 SLOT="0"
+
+DEPEND="virtual/rubygems"
