diff -Naur standalone/PkgDirCheck/MismatchedPN/MismatchedPN-1.ebuild fixed/PkgDirCheck/MismatchedPN/MismatchedPN-1.ebuild
--- standalone/PkgDirCheck/MismatchedPN/MismatchedPN-1.ebuild	1969-12-31 17:00:00.000000000 -0700
+++ fixed/PkgDirCheck/MismatchedPN/MismatchedPN-1.ebuild	2019-08-20 18:54:59.098260825 -0600
@@ -0,0 +1,4 @@
+DESCRIPTION="Ebuilds that have different names than their parent directory"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
+SLOT="0"
+LICENSE="BSD"
diff -Naur standalone/PkgDirCheck/MismatchedPN/Mismatched-1.ebuild fixed/PkgDirCheck/MismatchedPN/Mismatched-1.ebuild
--- standalone/PkgDirCheck/MismatchedPN/Mismatched-1.ebuild	2019-08-20 18:54:59.098260825 -0600
+++ fixed/PkgDirCheck/MismatchedPN/Mismatched-1.ebuild	1969-12-31 17:00:00.000000000 -0700
@@ -1,4 +0,0 @@
-DESCRIPTION="Ebuilds that have different names than their parent directory"
-HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-SLOT="0"
-LICENSE="BSD"
