diff -Naur eclass/EclassUsageCheck/MisplacedEclassVar/MisplacedEclassVar-0.ebuild fixed/EclassUsageCheck/MisplacedEclassVar/MisplacedEclassVar-0.ebuild
--- eclass/EclassUsageCheck/MisplacedEclassVar/MisplacedEclassVar-0.ebuild	2021-05-23 20:23:16.423009026 -0600
+++ fixed/EclassUsageCheck/MisplacedEclassVar/MisplacedEclassVar-0.ebuild	2021-05-23 20:23:43.734588313 -0600
@@ -4,8 +4,8 @@
 	inherit vcs
 fi
 
-inherit pre-inherit
 PRE_INHERIT_VAR="foo"
+inherit pre-inherit
 
 DESCRIPTION="Ebuild with misplaced pre-inherit eclass variable"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
