diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-0.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-0.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-0.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-0.ebuild	2023-03-21 17:27:31.612748202 +0100
@@ -3,10 +3,13 @@
 
 EAPI=8
 
+PYPI_NO_NORMALIZE=1
+PYPI_PN=${PN/-/.}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN/-/.}/${P/-/.}.tar.gz"
-S=${WORKDIR}/${P/-/.}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000.ebuild	2023-03-21 17:28:15.590371586 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN/-/.}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN/-/.}/${PN//-/_}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r100.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r100.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r100.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r100.ebuild	2023-03-21 17:28:22.750527209 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN/-/_}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN/-/_}/${PN//-/_}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r101.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r101.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r101.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r101.ebuild	2023-03-21 17:28:29.327336791 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN//-/_}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN//-/_}/${PN//-/_}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r1.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r1.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r1.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r1.ebuild	2023-03-21 17:28:35.514137872 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN//-/.}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN//-/.}/${PN//-/_}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r200.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r200.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r200.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1000-r200.ebuild	2023-03-21 17:28:42.704294057 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN^}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/P/${PN^}/${PN//-/_}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-100.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-100.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-100.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-100.ebuild	2023-03-21 17:28:57.301277709 +0100
@@ -3,10 +3,12 @@
 
 EAPI=8
 
+PYPI_PN=${PN/-/_}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN/-/_}/${P/-/_}.tar.gz"
-S=${WORKDIR}/${P/-/_}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-1.ebuild	2023-03-21 17:27:53.069881865 +0100
@@ -3,10 +3,13 @@
 
 EAPI=8
 
+PYPI_NO_NORMALIZE=1
+PYPI_PN=${PN//-/.}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN//-/.}/${PN//-/.}-${PV}.tar.gz"
-S=${WORKDIR}/${PN//-/.}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-2000.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-2000.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-2000.ebuild	2023-03-21 17:20:27.603389499 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-2000.ebuild	2023-03-21 17:30:27.623252678 +0100
@@ -3,13 +3,15 @@
 
 EAPI=8
 
+PYPI_PN=${PN/-/.}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with two PyPI URLs"
 HOMEPAGE="https://example.com"
-SRC_URI="
-	mirror://pypi/${PN::1}/${PN/-/.}/${PN//-/_}-${PV}.tar.gz
-	mirror://pypi/${PN::1}/${PN/-/.}_vectors/${PN/-/.}_vectors-151.tar.gz
+SRC_URI+="
+	$(pypi_sdist_url --no-normalize python.inline-pypi-url_vectors 151)
 "
-S=${WORKDIR}/${PN//-/_}-${PV}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-200.ebuild fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-200.ebuild
--- python/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-200.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/python-inline-pypi-url/python-inline-pypi-url-200.ebuild	2023-03-21 17:28:06.286835989 +0100
@@ -3,10 +3,13 @@
 
 EAPI=8
 
+PYPI_NO_NORMALIZE=1
+PYPI_PN=${PN^}
+
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/P/${PN^}/${P^}.tar.gz"
-S=${WORKDIR}/${P^}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-0.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-0.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-0.ebuild	2023-03-02 17:59:30.068345977 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-0.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,9 +3,11 @@
 
 EAPI=8
 
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.zip"
+SRC_URI="$(pypi_sdist_url --no-normalize "${PN}" "${PV}" .zip)"
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1.ebuild	2023-03-02 17:59:30.068345977 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,11 +3,11 @@
 
 EAPI=8
 
-MY_P=${P/_alpha/a}
+PYPI_NO_NORMALIZE=1
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN}/${MY_P}.tar.gz"
-S=${WORKDIR}/${MY_P}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1-r1.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1-r1.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1-r1.ebuild	2023-03-02 17:59:30.068345977 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-1_alpha1-r1.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,14 +3,14 @@
 
 EAPI=8
 
-MY_P=${P/_alpha/a}
+PYPI_NO_NORMALIZE=1
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="
-	mirror://pypi/${PN::1}/${PN}/${MY_P}.tar.gz
+SRC_URI+="
 	https://example.com/foo.patch
 "
-S=${WORKDIR}/${MY_P}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4.ebuild	2023-03-02 17:59:30.068345977 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,11 +3,10 @@
 
 EAPI=8
 
-MY_P=${PN,,}-${PV/_p/.post}
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="mirror://pypi/${PN::1}/${PN}/${MY_P}.tar.gz"
-S=${WORKDIR}/${MY_P}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r1.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r1.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r1.ebuild	2023-03-02 17:59:30.068345977 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r1.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,14 +3,13 @@
 
 EAPI=8
 
-MY_P=${PN,,}-${PV/_p/.post}
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
-SRC_URI="
-	mirror://pypi/${PN::1}/${PN}/${MY_P}.tar.gz
+SRC_URI+="
 	https://example.com/foo.patch
 "
-S=${WORKDIR}/${MY_P}
 
 LICENSE="BSD"
 SLOT="0"
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,13 +3,14 @@
 
 EAPI=8
 
+inherit pypi
+
 MY_P=pypi-url-${PV}
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
 SRC_URI="
-	mirror://pypi/p/pypi-url/${MY_P}.zip
-	https://files.pythonhosted.org/packages/py3/p/pypi-url/pypi_url-${PV}-py3-none-any.whl
-		-> pypi_url-${PV}-py3-none-any.whl.zip
+	$(pypi_sdist_url --no-normalize pypi-url "${PV}" .zip)
+	$(pypi_wheel_url --unpack pypi-url)
 "
 S=${WORKDIR}/${MY_P}
 
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3-r1.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3-r1.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3-r1.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-3-r1.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,12 +3,14 @@
 
 EAPI=8
 
+inherit pypi
+
 MY_P=pypi-url-${PV}
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
 SRC_URI="
-	mirror://pypi/p/pypi-url/${MY_P}.tar.gz
-	https://files.pythonhosted.org/packages/py3/p/pypi-url/pypi_url-3-py3-none-any.whl
+	$(pypi_sdist_url --no-normalize pypi-url)
+	$(pypi_wheel_url pypi-url)
 "
 S=${WORKDIR}/${MY_P}
 
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-4.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-4.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-4.ebuild	2023-03-02 17:59:30.071679391 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-4.ebuild	2023-03-21 17:26:48.258469809 +0100
@@ -3,12 +3,14 @@
 
 EAPI=8
 
+inherit pypi
+
 MY_P=pypi_url-${PV}
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
 SRC_URI="
-	mirror://pypi/p/pypi-url/${MY_P}.tar.gz
-	https://files.pythonhosted.org/packages/cp310/${PN::1}/${PN}/${P,,}-cp310-cp310-linux_x86_64.whl
+	$(pypi_sdist_url pypi-url)
+	$(pypi_wheel_url "${PN}" "${PV}" cp310 cp310-linux_x86_64)
 "
 S=${WORKDIR}/${MY_P}
 
diff -Naur python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r2.ebuild fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r2.ebuild
--- python/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r2.ebuild	2023-03-25 18:33:03.235669903 +0100
+++ fixed/PythonFetchableCheck/PythonInlinePyPIURL/PythonInlinePyPIURL-2_p4-r2.ebuild	2023-03-25 18:37:09.307850231 +0100
@@ -3,13 +3,14 @@
 
 EAPI=8
 
-MY_P=${PN,,}-${PV/_p/.post}
+PYPI_PN=${PN,,}
+inherit pypi
+
 DESCRIPTION="Ebuild with PyPI URL"
 HOMEPAGE="https://example.com"
 SRC_URI="
-	mirror://pypi/${PN::1}/${PN}/${MY_P}.tar.gz -> python-${MY_P}.tar.gz
+	$(pypi_sdist_url) -> python-${MY_P}.tar.gz
 "
-S=${WORKDIR}/${MY_P}
 
 LICENSE="BSD"
 SLOT="0"
