diff -Naur standalone/SrcUriCheck/SrcUriFilenameDotPrefix/Manifest fixed/SrcUriCheck//SrcUriFilenameDotPrefix/Manifest
--- standalone/SrcUriCheck/SrcUriFilenameDotPrefix/Manifest
+++ fixed/SrcUriCheck/SrcUriFilenameDotPrefix/Manifest
@@ -1 +1 @@
-DIST .gh.tar.gz 153310 BLAKE2B abcd4cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7995bcb7b255cdf810ce8cd16a957fb5bca3d1e71c088cd894968641db5dfae1c4c059df638 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
+DIST SrcUriFilenameDotPrefix-v0.gh.tar.gz 153310 BLAKE2B abcd4cd9bebe912f9c8877c0f09df059130c2dc5c4da8c926f8df7995bcb7b255cdf810ce8cd16a957fb5bca3d1e71c088cd894968641db5dfae1c4c059df638 SHA512 86ff9e1c4b9353b1fbb475c7bb9d2a97bd9db8421ea5190b5a84832930b34cb5b79f8c3da68a5eb8db334f06851ec129cc6611a371e47b7c5de7a615feec5e05
diff --git standalone/SrcUriCheck/SrcUriFilenameDotPrefix/SrcUriFilenameDotPrefix-0.ebuild fixed/SrcUriCheck/SrcUriFilenameDotPrefix/SrcUriFilenameDotPrefix-0.ebuild
--- standalone/SrcUriCheck/SrcUriFilenameDotPrefix/SrcUriFilenameDotPrefix-0.ebuild
+++ fixed/SrcUriCheck/SrcUriFilenameDotPrefix/SrcUriFilenameDotPrefix-0.ebuild
@@ -4,6 +4,6 @@ MY_P="${PN}-v${PV}"

 DESCRIPTION="Ebuild with empty var in SRC_URI"
 HOMEPAGE="https://github.com/pkgcore/pkgcheck"
-SRC_URI="https://github.com/pkgcore/pkgcheck/${MY_P}.tar.gz -> ${MYP}.gh.tar.gz"
+SRC_URI="https://github.com/pkgcore/pkgcheck/${MY_P}.tar.gz -> ${MY_P}.gh.tar.gz"
 LICENSE="BSD"
 SLOT="0"
